e2e/compose-env.yaml

```diff
@@ -10,12 +10,9 @@ services:
       command: ['--insecure-registry=registry:5000']
 
   notary-server:
-      image: 'notary:server-0.4.2'
+      build:
+        context: ..
```

Comment:
Looks like we only need `testdata` from this directory;

```Dockerfile
COPY e2e/testdata/notary/ /fixtures
```

May want to change this to

```
context: ./testdata/
```

And the Dockerfile to;

```
COPY ./notary/ /fixtures/
```

(or a variation thereof)